src/Microsoft.Identity.Web/Microsoft.Identity.Web.csproj

```diff
@@ -8,14 +8,20 @@
 
     <DefineConstants>$(DefineConstants);WEB</DefineConstants>
     <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
+    <!-- Copyright needs to be in the form of © not (c) to be compliant -->
+    <Title>Microsoft Identity Web</Title>
     <Authors>Microsoft</Authors>
-    <Company>Microsoft</Company>
-    <Description>This package enables ASP.NET Core Web apps and Web APIs to use the Microsoft identity platform (formerly Azure AD v2.0). When they call Web APIs, MSAL.NET is used to acquire tokens</Description>
+    <Company>Microsoft Corporation</Company>
+    <Product>Microsoft Identity Web</Product>
+    <Description>
+      This package enables ASP.NET Core Web apps and Web APIs to use the Microsoft identity platform (formerly Azure AD v2.0). 
+      This package is specifically used for web applications, which sign-in users, and protected web APIs, which optionally call downstream web APIs.
+    </Description>
     <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
     <PackageProjectUrl>https://github.com/AzureAD/microsoft-identity-web</PackageProjectUrl>
     <RepositoryUrl>https://github.com/AzureAD/microsoft-identity-web</RepositoryUrl>
-    <PackageTags>Microsoft identity web .NET ASP.NET Core</PackageTags>
+    <PackageTags>Microsoft identity web .NET ASP.NET Core Web App Web API B2C</PackageTags>
```

Comment:
Nit: Identity Web capitalized?

Reply:
fixed! nice catch

---
In reply to: [402046519](https://github.com/AzureAD/microsoft-identity-web/pull/75#discussion_r402046519) [](ancestors = 402046519)